library/ssl_srv.c

```diff
@@ -817,7 +817,8 @@ static int ssl_ciphersuite_match( mbedtls_ssl_context *ssl, int suite_id,
         return( MBEDTLS_ERR_SSL_INTERNAL_ERROR );
     }
 
-    MBEDTLS_SSL_DEBUG_MSG( 3, ( "trying ciphersuite: %s", suite_info->name ) );
+    MBEDTLS_SSL_DEBUG_MSG( 3, ( "trying ciphersuite: %04x (%s)",
```

Comment:
I'm sorry not to have annotated this before. The change here doesn't account for the fact of printing hexadecimal numbers. @hanno-arm could you please fix this as well? Or maybe I am missing a reason for the code being this way here?

Reply:
Fixed.